--- libe-book-0.1.2/src/lib/XMLTreeWalker.cpp.orig	2014-12-31 09:00:16.000000000 +0100
+++ libe-book-0.1.2/src/lib/XMLTreeWalker.cpp	2016-05-18 16:12:03.744754212 +0200
@@ -318,7 +318,7 @@
                              lwc_string *lang, bool *match);
 
   css_error css_node_presentational_hint(void *pw, void *node,
-                                         uint32_t property, css_hint *hint);
+                                         uint32_t *nhints, css_hint **hints);
 
   css_error css_ua_default_for_property(void *pw, uint32_t property,
                                         css_hint *hint);
@@ -1055,13 +1055,13 @@
     return CSS_OK;
   }
 
-  css_error css_node_presentational_hint(void *pw, void *const node, const uint32_t property, css_hint *const hint)
+  css_error css_node_presentational_hint(void *pw, void *const node, uint32_t *nhints, css_hint **const hints)
   {
     // TODO: implement me
     (void) pw;
     (void) node;
-    (void) property;
-    (void) hint;
+    (void) nhints;
+    (void) hints;
 
     return CSS_PROPERTY_NOT_SET;
   }
